### DISCUSSION TITLE
Possible to Ignore the Gemfile.lock in Shakapacker's root directory?

### DISCUSSION THREAD

**Discussion by RailsCod3rFuture** [shakacode/shakapacker#163] [Ideas]
Given that the gem lock file is inside of the root directory for the gem -- its causing AWS to pickup anomalies & things that it consider vulnerabilities/outdated gems. Is it mandatory that we have the gemfile.lock stored with the gem?

**Comment by tomdracz** [comment]
Thanks @RailsCod3rFuture - created an issue from this
